buildout.cfg

```diff
@@ -0,0 +1,4 @@
+[buildout]
+extends = https://raw.github.com/collective/buildout.plonetest/master/test-4.3.x.cfg
```

Comment:
@tomgross any plan to also add a 5.0.x and/or 5.x buildout as well? again p.a.tiles can be a good model to base on, as this is used by p.a.mosaic dependencies it would be good to know if it works on Plone 5 as well